.github/workflows/push.yml

```diff
@@ -54,11 +63,14 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
+          # Use the latest version of RubyGems with Ruby 3.0 to avoid:
+          # https://bugs.ruby-lang.org/issues/19371
+          rubygems: ${{ startsWith(matrix.ruby-version, '3.0') && 'latest' || 'default' }}
```

Comment:
i thought i was going crazy for a second because I was sure I'd already merged something like this in... but that was in #162 that targeted the `v2` branch 😅

Reply:
Yup yup I decided it's good to have it in master as well 😄